### PR TITLE
Fix incremental update longrun 6 14 z

### PR DESCRIPTION
**Pull request by vijaysawant**
### Problem Statement
Hosts-Content > longrun > increamental update feature test was failing in 6.13.z & 6.14.z because content host unabled to install katello-host-tools

Error - `failed on setup with "robottelo.hosts.ContentHostError: Failed to install katello-host-tools"`

### Solution
Implemented changes which will solve said problem

### Related Issues
Same problem has been seen in 6.13.z as well

### CODE REVIEW

**Comment by vijaysawant** [comment]
corrected target branch

**Comment by vijaysawant** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_inc_updates.py -k test_positive_noapply_api

File: tests/foreman/longrun/test_inc_updates.py [hunk 1]
```diff
@@ -207,28 +213,25 @@ def test_positive_noapply_api(
...
-    # Get the content view versions and use the recent one. API always
-    # returns the versions in ascending order (last in the list is most recent)
-    cv_versions = module_cv.version
     # Get the applicable errata
     errata_list = get_applicable_errata(custom_repo)
     assert len(errata_list) > 0
 
     # Apply incremental update using the first applicable errata
-    entities.ContentViewVersion().incremental_update(
+    outval = entities.ContentViewVersion().incremental_update(
```
Comment (jameerpathan111) [inline tests/foreman/longrun/test_inc_updates.py:222]: There are some inconsistencies in use of `module_target_sat` fixture, I see the `host` fixture is using it, but the test is using `entities`(also `module_cv` fixture`). It's probably good to address this now.
```suggestion
    outval = module_target_sat.ContentViewVersion().incremental_update(
```

File: tests/foreman/longrun/test_inc_updates.py [hunk 2]
```diff
@@ -207,28 +213,25 @@ def test_positive_noapply_api(
...
+                    'content_view_version_id': cvv.id,
                     'environment_ids': [dev_lce.id],
                 }
             ],
             'add_content': {'errata_ids': [errata_list[0].id]},
         }
     )
-    # Re-read the content view to get the latest versions
-    module_cv = module_cv.read()
-    assert len(module_cv.version) > len(cv_versions)
```
Comment (jameerpathan111) [inline tests/foreman/longrun/test_inc_updates.py:234]: Can you explain why this assertion is not good and is being removed?

File: tests/foreman/longrun/test_inc_updates.py [hunk 3]
```diff
@@ -66,19 +66,19 @@ def qe_lce(module_entitlement_manifest_org, dev_lce):
...
+def sat_client_repo(module_entitlement_manifest_org, module_target_sat):
     """Enable Sat tools repository"""
-    rhel7_sat6tools_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+    sat_client_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
         basearch=DEFAULT_ARCHITECTURE,
         org_id=module_entitlement_manifest_org.id,
         product=PRDS['rhel'],
-        repo=REPOS['rhst7']['name'],
-        reposet=REPOSET['rhst7'],
+        repo=REPOS['rhsclient7']['name'],
```
Comment (jameerpathan111) [inline tests/foreman/longrun/test_inc_updates.py:75]: Would it make more sense to test rhel8 or rhel9 instead? Non-blocking just throwing ideas:)

**Review by jameerpathan111 — CHANGES_REQUESTED** [review]

File: tests/foreman/longrun/test_inc_updates.py (hunk 1 above)
Comment (vijaysawant) [reply tests/foreman/longrun/test_inc_updates.py:222]: replacement of `entities` with `module_target_sat.api` will be taken care by @omkarkhatavkar in framework improvement tasks. So I am not changing non affected part of test module.

**Review by vsedmik — APPROVED** [review]

**Review by jameerpathan111 — APPROVED** [review]

File: tests/foreman/longrun/test_inc_updates.py (hunk 2 above)
Comment (vijaysawant) [reply tests/foreman/longrun/test_inc_updates.py:234]: Yea, there was reported BZ for unsorted cv version, In this test module we are ensuring api increamental update of applicable errata working fine without error, we are able to assert this change.

Comment (vijaysawant) [reply tests/foreman/longrun/test_inc_updates.py:234]: Hope so I have answered your query.

**Pull request merged**